### DISCUSSION TITLE
gold knob behaviour when jumping in & out of clips

### DISCUSSION THREAD

**Discussion by duncanrmi** [SynthstromAudible/DelugeFirmware#3178] [Ideas]
quite often.... all the time, in fact..... I'll go into a clip to adjust something- cutoff, level, pan, whatever.... & then want to do the same adjustment on another clip. 
default behaviour is for the UI to remember what the last thing was that you edited in each clip, & show you that. 
it would be far more useful to me, not to say more intuitive, if the selection in the gold knob area would persist from one clip to the next. it's slowing me down atm & causing errors.

make sense?

**Comment by nikodemus** [comment]
I agree, esp. when edits are done from grid view without entring the clip.
